### PR TITLE
Change description of --installed option for "dnf repoquery"

### PR DESCRIPTION
**Pull request by j-mracek**

### CODE REVIEW

File: doc/repoquery.rst [hunk 1]
```diff
@@ -64,7 +64,8 @@ Together with ``<pkg-spec>``, control what packages are displayed in the output.
     Limit the resulting set only to package that owns ``<file>``.
 
 ``--installed``
-    Limit the resulting set to installed packges.
+    Limit the resulting set to installed packges. If in dnf.conf file exclude
+    option is used, the option \-\ `-disableexcludes\ <https://dnf.readthedocs.org/en/latest/command_ref.html#options-label>`_ ensure that all installed packages will be listed.
```
Comment (jsilhan) [inline doc/repoquery.rst]: the **command line** option

Comment (jsilhan) [inline doc/repoquery.rst]: please, document it in `command_ref.rst` for `dnf list installed` (with references to disableexcludes) in dnf too.

**Comment by j-mracek** [comment]
I updated the description.

**Comment by j-mracek** [comment]
I added another commit for enhancement of --add-repo option. It should be not merged before installroot PR will be merged.

File: doc/repoquery.rst [hunk 2]
```diff
@@ -64,7 +64,10 @@ Together with ``<pkg-spec>``, control what packages are displayed in the output.
     Limit the resulting set only to package that owns ``<file>``.
 
 ``--installed``
-    Limit the resulting set to installed packges.
+    Limit the resulting set to installed packages. The `exclude <https://dnf.readthedocs.org/en/latest/conf_ref.html#exclude-label>`_
+    option in configuration file (.conf) might influence the result, but if the
+    command line option \-\ `-disableexcludes\ <https://dnf.readthedocs.org/en/latest/command_ref.html#disableexcludes-label>`_
+    is used, it ensure that all installed packages will be listed.
```
Comment (mluscon) [inline doc/repoquery.rst]: ensures

**Comment by mluscon** [comment]
merged

**Pull request closed without merging**